internal/criocli/criocli.go

```diff
@@ -640,6 +643,11 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			Usage:   "Socket for the metrics endpoint",
 			EnvVars: []string{"CONTAINER_METRICS_SOCKET"},
 		},
+		&cli.StringFlag{
+			Name:    "big-files-temporary-dir",
+			Usage:   "Temporary directory to use for storing big files",
```

Comment:
I'd like to see more docs here. Maybe a sentence about "big files".

Reply:
Personally I think this is fine as is, however, I would very much love to see what @saschagrunert is asking for here to be added to the man pages.

Reply:
I think it can be improved by something like 
```
Path to the temporary directory to use for storing big files, used to store image blobs and data streams related to containers image management.
```